### ISSUE TITLE
VLC output (-o vlc): Wrong charset in debug output

### ISSUE THREAD

**Issue by mighty-p** [mighty-p/t2scan#11]
When using "-o vlc", t2scan uses automatically iso8859-1 output, which creates on an utf-8 system debug like this:

```
service = Radio �sterreich 1 (ORF)
service = Hitradio �3 (ORF)
service = radio FM4 (ORF)
service = �3-Visual-Radio (ORF)
```

Thanks @andimik for reporting this within Issue #8 .

**Comment by mighty-p** [comment]
should be fixed by commit 7d35e17.

**Comment by andimik** [comment]
Yes, this is fixed now.

```
andreas@DellMATE:~/apps/t2scan$ ./t2scan -t 2 -l 24 -p 1 -o vlc > /tmp/test.xspf./t2scan -t 2 -l 24 -p 1 -o vlc 
t2scan version 20200517 (compiled for DVB API 5.11)
guessing country 'DE', use -Y <country> to override
using settings for GERMANY
DVB aerial
DVB-T Europe UHF <790MHz
scan type TERRESTRIAL, channellist 16
output format vlc xspf playlist
output charset 'UTF-8', use -I <charset> to override
Info: using DVB adapter auto detection.
	/dev/dvb/adapter0/frontend0 -> TERRESTRIAL "Sony CXD2820R": very good :-))

Using TERRESTRIAL frontend (adapter /dev/dvb/adapter0/frontend0)
-_-_-_-_ Getting frontend capabilities-_-_-_-_ 
Using DVB API 5.11
frontend 'Sony CXD2820R' supports
DVB-T2
INVERSION_AUTO
QAM_AUTO
TRANSMISSION_MODE_AUTO
GUARD_INTERVAL_AUTO
HIERARCHY_AUTO
FEC_AUTO
BANDWIDTH_AUTO not supported, trying 6/7/8 MHz.
MULTISTREAM
FREQ (45.00MHz ... 864.00MHz)
-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_ 
Scanning DVB-T2...
Scanning 8MHz frequencies...
498000 (CH24): (time: 00:00.600) 
   plp id 1:   signal ok:	QAM_AUTO f = 498000 kHz I999B8C999D999T999G999Y999P1 (0:0:0)
        QAM_AUTO f = 498000 kHz I999B8C999D0T999G999Y0P1 (8232:13104:401) : scanning for services
	service = ORF1 HD (ORF)
	service = ORF2 T HD (ORF)
	service = ORF2 V HD (ORF)
	service = ORF2 K HD (ORF)
	service = ORF III HD (ORF)
	service = ORF Sport+ HD (ORF)
	service = ORF1 (ORF)
	service = ORF2 W (ORF)
	service = Radio Österreich 1 (ORF)
	service = Hitradio Ö3 (ORF)
	service = radio FM4 (ORF)
	service = Ö3-Visual-Radio (ORF)
	service = SRT8505 OTA (ORS)
(time: 00:06.184) dumping lists (12 services)
..
Done, scan time: 00:06.184
andreas@DellMATE:~/apps/t2scan$ 
```
[test.xspf.txt](https://github.com/mighty-p/t2scan/files/4648564/test.xspf.txt)
(Github cannot attach xspf files, so changed to txt)

![grafik](https://user-images.githubusercontent.com/24510556/82291869-11c78580-99aa-11ea-89a1-aa33e062bf25.png)

![grafik](https://user-images.githubusercontent.com/24510556/82292155-9d411680-99aa-11ea-9661-56cb2ced3800.png)


It's playing well in VLC (even due to the fact that the ID does not really match now), you may close this issue.
